cli.js

```diff
@@ -4,12 +4,23 @@ const publish = require('./src/publish')
 const yargs = require('yargs')
   .option('dry-run', {
     describe: 'Print what will be done without doing it',
+    default: process.env.INPUT_DRY_RUN || false,
```

Comment:
Is this smart enough to turn the string "false" into a boolean false? I can't remember if we tested that.

Reply:
I believe we did test that but I will double check. Is the idea that even if someone passes in a boolean it might end up converted back into a string once it gets to process.env?

Reply:
Confirmed in the `test-package` repo that it does convert a string `"false"` to a boolean :)